site/src/modules/workspaces/DynamicParameter/DynamicParameter.tsx

```diff
@@ -458,7 +467,7 @@ const ParameterField: FC<ParameterFieldProps> = ({
 					<Slider
 						id={id}
 						className="mt-2"
-						value={[Number(value)]}
+						value={[Number.isNaN(Number(value)) ? 0 : Number(value)]}
```

Comment:
I would recommend using `Number.isFinite` here, which handles `NaN` but also `Infinity` and `-Infinity`
```suggestion
						value={[Number.isFinite(Number(value)) ? Number(value) : 0]}
```